frontend/packages/kubevirt-plugin/src/components/virtualization-overview/permissions-card/PermissionsCardPopover.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react';
+import { Button, Popover, PopoverPosition } from '@patternfly/react-core';
+import { useTranslation } from 'react-i18next';
+import { ModalBody, ModalTitle } from '@console/internal/components/factory';
+import { GreenCheckCircleIcon, YellowExclamationTriangleIcon } from '@console/shared';
+
+import './permissions-card-popover.scss';
+
+type PermissionItemProps = {
+  task: string;
+  capability: boolean;
+  isLoading: boolean;
+};
+
+type PermissionsCardPopoverProps = {
+  capabilitiesData: { [key: string]: boolean };
+};
+
+const PermissionItem: React.FC<PermissionItemProps> = ({ task, capability, isLoading }) => {
```

Comment:
its a nested component, consider extracting to a different file :)

Reply:
Done!